tests/functional/test_install_wheel.py

```diff
@@ -587,3 +588,18 @@ def test_wheel_install_fails_with_badly_encoded_metadata(script):
     assert "Error decoding metadata for" in result.stderr
     assert "simple-0.1.0-py2.py3-none-any.whl" in result.stderr
     assert "METADATA" in result.stderr
+
+
+@pytest.mark.parametrize(
+    'package_name',
+    ['simple-package', 'simple_package'],
+)
+def test_correct_package_name_while_creating_wheel_bug(script, package_name):
```

Comment:
IMHO, we do not want functional tests for our testing utility functions.
Possibly an unit test checking that the generated wheel filename has been normalized would be enough.

Reply:
I have updated the test to just check the generated wheel filename